tools/nnyeah/nnyeah/nnyeah.csproj

```diff
@@ -5,6 +5,14 @@
     <TargetFramework>net5.0</TargetFramework>
     <RootNamespace>Microsoft.MaciOS.Nnyeah</RootNamespace>
     <Nullable>enable</Nullable>
+    <IsPackable>true</IsPackable>
+    <PackAsTool>true</PackAsTool>
+    <ToolCommandName>dotnet-nnyeah</ToolCommandName>

```

Comment:
we could chante the name of the tool now, even if the namespace and .exe are nnyeah. @dalexsoto ?